packages/cubejs-sqlite-driver/driver/index.d.ts

```diff
@@ -1,9 +1,10 @@
+import { BaseDriver } from "@cubejs-backend/query-orchestrator";
 declare module "@cubejs-backend/sqlite-driver" {
   interface SqliteOptions {
     database: string;
   }
 
-  export default class SqliteDriver {
+  export default class SqliteDriver extends BaseDriver{
```

Comment:
Missing space after `BaseDriver`

```suggestion
  export default class SqliteDriver extends BaseDriver {
```